clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/AbstractEnableDisableAtomicOperation.groovy

```diff
@@ -108,26 +108,30 @@ abstract class AbstractEnableDisableAtomicOperation implements AtomicOperation<V
         asgService.resumeProcesses(serverGroupName, AutoScalingProcessType.getDisableProcesses())
       }
 
-      List<String> instanceIds = new ArrayList<>()
-      DescribeInstancesRequest describeInstancesRequest = new DescribeInstancesRequest().withFilters(
-        new Filter().withName(INSTANCE_ASG_TAG_NAME).withValues(serverGroupName)
-      )
-
-      DescribeInstancesResult describeInstancesResult = regionScopedProvider.amazonEC2.describeInstances(describeInstancesRequest)
-      List<Reservation> reservations = describeInstancesResult.getReservations()
-      while (describeInstancesResult.getNextToken()) {
-        describeInstancesRequest.setNextToken(describeInstancesResult.getNextToken())
-        describeInstancesResult = regionScopedProvider.amazonEC2.describeInstances(describeInstancesRequest)
-        reservations += describeInstancesResult.getReservations()
-      }
-
-      for (Reservation reservation : reservations) {
-        instanceIds += reservation.getInstances().findAll {
-          [ InstanceStateName.Running, InstanceStateName.Pending ].contains(InstanceStateName.fromValue(it.getState().getName()))
-        }*.instanceId
+      List<String> instanceIds = asg.instances.findAll {
```

Comment:
Do we need to filter the list of instances since we are doing it after the describe-instances?